### PR TITLE
5252.dev

### PR DESCRIPTION
**Pull request by YaroslavTheSlav**
Weather Module

### CODE REVIEW

File: web/modules/slavnews_weather/slavnews_weather.install [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+ */
+function slavnews_weather_uninstall() {
+  \Drupal::messenger()->addStatus(__FUNCTION__);
+}
+
+/**
+ * Implements hook_schema().
+ */
+function slavnews_weather_schema() {
+  $schema['slavnews_weather_example'] = [
```
Comment (boddy-devbranch) [inline web/modules/slavnews_weather/slavnews_weather.install]: Could you tell me please the purpose of this schema?

File: web/modules/slavnews_weather/slavnews_weather.install [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+/**
+ * Implements hook_requirements().
+ */
+function slavnews_weather_requirements($phase) {
+  $requirements = [];
+
+  if ($phase == 'runtime') {
+    $value = mt_rand(0, 100);
+    $requirements['slavnews_weather_status'] = [
+      'title' => t('SlavNews Weather status'),
```
Comment (boddy-devbranch) [inline web/modules/slavnews_weather/slavnews_weather.install]: WAT

File: web/modules/slavnews_weather/src/Plugin/Block/WeatherBlockFolder/WeatherBlock.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Drupal\slavnews_weather\Plugin\Block\WeatherBlockFolder;
+
+use Drupal\Core\Block\BlockBase;
+// phpcs:ignore
```
Comment (boddy-devbranch) [inline web/modules/slavnews_weather/src/Plugin/Block/WeatherBlockFolder/WeatherBlock.php]: what is it?

**Review by boddy-devbranch — CHANGES_REQUESTED** [review]

File: web/modules/slavnews_weather/slavnews_weather.install (hunk 2 above)
Comment (YaroslavTheSlav) [reply web/modules/slavnews_weather/slavnews_weather.install]: removed (that were default drush generated code)

File: web/modules/slavnews_weather/src/Plugin/Block/WeatherBlockFolder/WeatherBlock.php (hunk 3 above)
Comment (YaroslavTheSlav) [reply web/modules/slavnews_weather/src/Plugin/Block/WeatherBlockFolder/WeatherBlock.php]: fixed(replaced couple lines of code)

File: web/modules/slavnews_weather/slavnews_weather.install (hunk 1 above)
Comment (YaroslavTheSlav) [reply web/modules/slavnews_weather/slavnews_weather.install]: removed (that was default drush generated code)

**Review by boddy-devbranch — APPROVED** [review]

**Pull request merged**